### PR TITLE
fix(react-native): fix react-native upgrade

### PR DESCRIPTION
**Pull request by PlayMa256**

### CODE REVIEW

**Comment by PlayMa256** [comment]
Can someone test for IOS? I don't have a mac to run it!

File: packages/app/android/app/build.gradle [hunk 1]
```diff
@@ -93,30 +97,67 @@ def enableSeparateBuildPerCPUArchitecture = false
...
-    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 
     defaultConfig {
-        applicationId "com.reactnavigationrelay"
+        applicationId "com.rndiffapp"
```
Comment (jgcmarins) [inline packages/app/android/app/build.gradle]: should we keep `reactnavigationrelay`?

File: packages/app/android/app/src/main/java/com/reactnavigationrelay/MainApplication.java [hunk 2]
```diff
@@ -1,14 +1,16 @@
-package com.reactnavigationrelay;
+package com.rndiffapp;
```
Comment (jgcmarins) [inline packages/app/android/app/src/main/java/com/reactnavigationrelay/MainApplication.java]: `reactnavigationrelay`?

File: packages/app/android/app/build.gradle (hunk 1 above)
Comment (PlayMa256) [reply packages/app/android/app/build.gradle]: OHHH, so rndiffapp would be sort of the name of the application?

Comment (jgcmarins) [reply packages/app/android/app/build.gradle]: yep

Comment (PlayMa256) [reply packages/app/android/app/build.gradle]: Let me change it!!!

**Comment by PlayMa256** [comment]
@jgcmarins @sibelius i suppose it is correct now!

**Pull request merged**